doc/nrf/device_guides/nrf52/fota_update.rst

```diff
@@ -96,7 +96,7 @@ See how to build the :ref:`peripheral_lbs` sample with MCUboot in the direct-xip
 
```

Comment:
In the build command above:
`-DSB_CONFIG_BOOT_DIRECT_XIP=y` -> `-DSB_CONFIG_MCUBOOT_MODE_DIRECT_XIP=y`